src/dash-table/dash/DataTable.js

```diff
@@ -870,13 +877,11 @@ export const propTypes = {
             direction: PropTypes.oneOf(['asc', 'desc']).isRequired
         })),
 
-    /**
-     * If False, then empty strings (`''`) are considered
-     * valid values (they will appear first when sorting ascending).
-     * If True, empty strings will be ignored, causing these cells to always
-     * appear last.
-     */
-    sorting_treat_empty_string_as_none: PropTypes.bool,
+    sort_as_none: PropTypes.arrayOf(PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.number,
+        PropTypes.bool
+    ])),
```

Comment:
Like `locale_format`, provide a table-level `sort_as_none` that can be overridden by columns